.github/workflows/publish-discord-changelog.yml

```
@@ -0,0 +1,46 @@
+  workflow_run:
+    workflows: ["Publish Stable"]
+    types: [completed]
+    branches: [stable]
+
+permissions:
+  actions: read
+  contents: read
+
+concurrency:
+  group: publish-discord-changelog
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)